manage_breast_screening/notifications/tests/management/commands/test_create_appointments.py

```diff
@@ -16,7 +16,9 @@
 class TestCreateAppointments:
     @pytest.fixture
     def raw_data(self):
-        return open(f"{os.path.dirname(os.path.realpath(__file__))}/test.dat").read()
+        return open(
+            f"{os.path.dirname(os.path.realpath(__file__))}/ABC_20241202091221_APPT_106.dat"
```

Comment:
please could you rebase against main, as I've added a `test_updated.dat` file which should probably follow the same naming convention if this is what we are using! maybe it should just be 1 number/hour off this one or something?